internal/testutil/harnessutil/harnessutil.go

```diff
@@ -340,6 +345,8 @@ func parseHarnessOption(t *testing.T, key string, value any, options *HarnessOpt
 		options.CaptureSuggestions = value.(bool)
 	case "typescriptVersion":
 		options.TypescriptVersion = value.(string)
+	case "strictBuiltinIteratorReturn":
+		options.StrictBuiltinIteratorReturn = value.(bool)
```

Comment:
This shouldn't be required; all of these ones here are special ones added only for the harness. Are we just missing the parser code for this elsewhere? As in, is the `parsinghelpers.go` change sufficient?

Reply:
Yeah, checked, and reverting this + the struct change still allows the tests to pass.